data/data/ovirt/variables-ovirt.tf

```diff
@@ -50,12 +44,13 @@ variable "ovirt_storage_domain_id" {
   description = "The ID of Storage Domain for the template"
 }
 
-variable "openstack_base_image_name" {
+variable "ovirt_base_image_name" {
```

Comment:
Since  you are renaming this variable here, could you please consistently rename it everywhere?